omod/pom.xml

```diff
@@ -218,6 +219,7 @@
                     <execution>
                         <goals>
                             <goal>exec</goal>
+                        
```

Comment:
For consistency, you wouldn't want this white space here

Reply:
Sure @ODOROA am still working on this ticket and i hope to squash with new changes today thanks